strangelove-ventures/sommelier
# feat(ui): add ability to import somm reward token to metamask

Closes #252 

## Description

Add Ability to import somm reward token to metamask 

## Changes

- [x] [feat(ui): add ability to import somm reward token to metamask](https://github.com/strangelove-ventures/sommelier/commit/9c209fcc26a22039dddc467ed33a534d53ca923c)

## Screenshots: 

![CleanShot 2022-08-30 at 01 48 16](https://user-images.githubusercontent.com/39829726/187276249-96ef0084-39e6-4e46-919a-a9525f4167a2.gif)

![CleanShot 2022-08-31 at 01 44 08](https://user-images.githubusercontent.com/39829726/187517806-4e0d51a9-a931-4aae-a546-ab17384ea340.png)


## Testing Steps

1. if you already have somm token on metamask remove somm token on metamask
2. connect wallet
3. go to cellar page
4. click metamask button icon on the "Rewards" card
5. You should be able to see somm on metamask
